src/main.rs

```diff
@@ -4,100 +4,105 @@
-
 
 #[macro_use]
 extern crate nom;
 
-mod vwmap;
-mod parser;
-mod model_instance;
-mod feature_buffer;
-mod regressor;
-mod cmdline;
-mod cache;
-mod persistence;
-mod serving;
-mod optimizer;
-mod version;
-mod consts;
 mod block_ffm;
-mod block_lr;
-mod block_loss_functions;
 mod block_helpers;
-mod multithread_helpers;
-mod feature_transform_parser;
+mod block_loss_functions;
+mod block_lr;
+mod cache;
+mod cmdline;
+mod consts;
+mod feature_buffer;
 mod feature_transform_executor;
 mod feature_transform_implementations;
+mod feature_transform_parser;
+mod model_instance;
+mod multithread_helpers;
+mod optimizer;
+mod parser;
+mod persistence;
+mod regressor;
+mod serving;
+mod version;
+mod vwmap;
 
 fn main() {
     match main2() {
-        Err(e) => {println!("Global error: {:?}", e); std::process::exit(1)},
+        Err(e) => {
+            println!("Global error: {:?}", e);
+            std::process::exit(1)
+        }
         Ok(()) => {}
-    }    
+    }
 }
 
-fn main2() -> Result<(), Box<dyn Error>>  {
+fn main2() -> Result<(), Box<dyn Error>> {
     // We'll parse once the command line into cl and then different objects will examine it
     let cl = cmdline::parse();
 
     // Where will we be putting perdictions (if at all)
     let mut predictions_file = match cl.value_of("predictions") {
         Some(filename) => Some(BufWriter::new(File::create(filename)?)),
-        None => None      
+        None => None,
     };
 
     let testonly = cl.is_present("testonly");
 
-
     let final_regressor_filename = cl.value_of("final_regressor");
     match final_regressor_filename {
         Some(filename) => {
             if !cl.is_present("save_resume") {
                 return Err("You need to use --save_resume with --final_regressor, for vowpal wabbit compatibility")?;
             }
             println!("final_regressor = {}", filename);
-        },
+        }
         None => {}
     };
 
     let inference_regressor_filename = cl.value_of("convert_inference_regressor");
     match inference_regressor_filename {
         Some(filename1) => {
             println!("inference_regressor = {}", filename1);
-        },
+        }
         None => {}
     };
-    
-    
+
     /* setting up the pipeline, either from command line or from existing regressor */
     // we want heal-allocated objects here
-    
 
     if cl.is_present("daemon") {
-        let filename = cl.value_of("initial_regressor").expect("Daemon mode only supports serving from --initial regressor");
+        let filename = cl
+            .value_of("initial_regressor")
+            .expect("Daemon mode only supports serving from --initial regressor");
         println!("initial_regressor = {}", filename);
-        println!("WARNING: Command line model parameters will be ignored");
-        let (mi2, vw2, re_fixed) = persistence::new_regressor_from_filename(filename, true)?;
+		println!("WARNING: Selected cmd parameters will be updated.");
```

Comment:
basically now the user has no clue what will happen ... which parameters are going to be used?
please, think how to create a behavior that will be more clear. 
at very minimum there should be clear messaging about exactly which parameters are considered. 
the best case would be to actually prevent errors by complaining when user is trying to change parameters that are not supported. But that might be too big of a project. 

So at very least think about how to reduce confusion about what's happening...